code/modules/mining/fulton.dm

```diff
@@ -35,6 +35,7 @@ var/list/total_extraction_beacons = list()
 		if(!A)
```

Comment:
you mean `possible_beacons.len != 1`, but I don't really think always showing the input is a problem